scripts/eval/yamls/mpt_eval.yaml

```diff
@@ -1,4 +1,4 @@
-max_seq_len: 2048
+max_seq_len: 1024
```

Comment:
why this change?

Reply:
this is the standard length used for our gauntlet evals